cobbler/modules/sync_post_wingen.py

```diff
@@ -47,7 +47,7 @@ def register() -> Optional[str]:
     """
     if not HAS_HIVEX:
         logging.info(
-            "python3-hivex not found. If you need Automatic Windows Installation support, please install."
+            "python3-hivex or python3-pefile not found. If you need Automatic Windows Installation support, please install."
```

Comment:
I think it is reasonable to either rename the variable to something like `HAS_WINDOWS_DEPS` or to split the check and have two different error messages.

Reply:
Looking the code, it seems like both pefile and hivex are only used in certain circumstances.  Can we delay the logging/error until they are actually used, or is that "too late"?

Reply:
I would like to keep it there as this is something the user should know as early as possible to take appropriate action in my opinion.